changelog/8.6.1.yaml

```
@@ -0,0 +1,14 @@
+version: 8.6.1
+entries:
+    - kind: bug-fix
+      summary: Fix issue where its possible for a component to receive a unit without a config
```

```suggestion
      summary: Fix issue where Beats started by agent may fail with an 'output unit has no config' error.
```

Let's reword this to use the exact same language that is used in the known issue I put in the 8.6.0 release notes: https://www.elastic.co/guide/en/fleet/current/release-notes-8.6.0.html